### ISSUE TITLE
Uninformative error message in fit_generator when FullbatchNodeGenerator's flow method has no train labels passed to it.

### ISSUE THREAD

**Issue by habiba-h** [stellargraph/stellargraph#407]
### Description

Calling `fit_generator` with a `FullBatchGenerator.flow` method that has no train labels passed to it results in a long and uninformative error message that is not caught.   

### Done Checklist (Bug)
- [ ] Bug fixed
- [ ] Branch and Pull Request build on CI
- [ ] Branch and Pull Request pass unit tests on CI
- [ ] Branch and Pull Request pass integration tests on CI
- [ ] Version number reflects new status
- [ ] Peer Code Review Performed
- [ ] Code well commented
- [ ] CHANGELOG.md updated

**Comment by youph** [comment]
@habiba-h good catch! Could you please add "how to reproduce" in the description?
